client/rule_test.go

```diff
@@ -161,4 +166,169 @@ func TestRules(t *testing.T) {
 		}
 		<-time.After(time.Millisecond * 10)
 	}
+
+	//test for Disabled function
+	//Condition Disabled Test
+	c = client.Condition{
+		ID:            "ID-condition",
+		Parent:        r.ID,
+		Description:   "cond vin high",
+		ConditionType: data.PointValuePointValue,
+		PointType:     data.PointTypeValue,
+		ValueType:     data.PointValueOnOff,
+		NodeID:        vin.ID,
+		Operator:      data.PointValueEqual,
+		Value:         0,
+		Disabled:      true,
+	}
+
+	err = client.SendNodeType(nc, c, "test")
+	if err != nil {
+		t.Fatal("Error sending node: ", err)
+	}
+
+	// wait for rule to get set up
+	time.Sleep(250 * time.Millisecond)
+
+	// set vin and look for vout to change
+	err = client.SendNodePoint(nc, vin.ID, data.Point{Type: data.PointTypeValue,
+		Value: 1, Origin: "test"}, true)
+	if err != nil {
+		t.Errorf("Error sending point: %v", err)
+	}
+
+	start = time.Now()
+	for {
+		if voutGet().Value == 1 {
+			// all is well
+			break
+		}
+		if time.Since(start) > time.Second {
+			t.Fatal("Timeout waiting for vout to be set")
+		}
+		<-time.After(time.Millisecond * 10)
+	}
+
+	//Action Disabled Test
+	c = client.Condition{
+		ID:            "ID-condition",
+		Parent:        r.ID,
+		Description:   "cond vin high",
+		ConditionType: data.PointValuePointValue,
+		PointType:     data.PointTypeValue,
+		ValueType:     data.PointValueOnOff,
+		NodeID:        vin.ID,
+		Operator:      data.PointValueEqual,
+		Value:         0,
+	}
```

Comment:
above comments apply to all the of the nodes created here and below -- create them once at the start of the test, and then only change the disabled point and var values as needed.